playbooks/kubevirt_web_ui.yml

```diff
@@ -0,0 +1,10 @@
+---
+- import_playbook: initial_configuration.yml
+
+- name: Kubevirt Web UI
+  hosts: localhost
+  connection: local
+  gather_facts: False
```

Comment:
You're missing:
``` yaml
  environment:
    http_proxy: ""
````

I think that might be causing the login failure.

Reply:
Added this to the playbook but the issue remains. I will keep trying.